docs/meshstack.identity-federation.md

```diff
@@ -3,7 +3,7 @@ id: meshstack.identity-federation
 title: Identity and Access Management
 ---
 
-One of meshStacks main features is to provide single identities across multiple cloud and container platforms. Therefore, meshStack contains a full-featured **Identity Broker** (meshIdB) which consumes identities from company directories and adds authorization information for the specific cloud platforms.
+meshStack supports a central Identity Provider architecture across multiple cloud and container platforms. Therefore, meshStack contains a full-featured Identity Broker (meshIdB) which consumes identities from company directories and adds authorization information for the specific cloud platforms.
```

Comment:
meshIdB adding authorization info is only relevant to the deprecated meshIdB based K8s, OpenStack, AWS integrations. 

I think it makes most sense to redesign this whole page and remove `meshStack-provisioned Identities` as a concept from the documentation.

Reply:
I agree that the page needs a general redesign. For the sake of getting important improvements on the page we will do the overhaul another time